### PR TITLE
✨ snek collection stats

### LINKED ISSUE #3207: History of floor and ceiling prices in BSX collection
We should show up floor and ceiling prices in BSX

### PR DESCRIPTION
**Pull request by roiLeo**
ℹ️ waiting for snek deploy

### PR type

- [x] Feature

### What's new?

- [x] PR closes #3207 
- [x] Requires deployment kodadot/snek#95
- [x] Requires deployment kodadot/snek#93
- [x] Requires deployment kodadot/snek#105
- [x] show collectionActivity on `/bsx/collection/{id}` & `/snek/collection/{id}`

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb)

### Screenshot

Screenshot 2022-10-10 at 08-45-40 kittys KodaDot - Kusama NFT Market Explorer

### CODE REVIEW

**Comment by yangwao** [comment]
@vikiival 👀

**Comment by vikiival** [comment]
Thanks for ping sir! Will resolve in the morning!

File: components/rmrk/Gallery/CollectionActivity.vue [hunk 1]
```diff
@@ -193,6 +193,7 @@ export default class CollectionActivity extends mixins(PrefixMixin) {
           id: this.id,
         },
       })
+      console.log(data)
```
Comment (roiLeo) [inline components/rmrk/Gallery/CollectionActivity.vue:196]:
```suggestion
```

**Comment by roiLeo** [comment]
Looks good!
Screenshot 2022-10-28 at 11-27-02 kittys KodaDot - Kusama NFT Market Explorer

**Comment by yangwao** [comment]
yessss
pay 150

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $150 @ 34.8 USD/KSM ~ 4.31 $KSM
🧗 [DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb](https://kusama.subscan.io/account/DVYy1qnocE8t6ZvUfPx3rEjG829khNRXx3YrCGVHHj19Lcb)
🔗 [0x7bd468eeaa0ada089e3415dc327f61aacda59aa824c90a74ce7b19933febed6a](https://kusama.subscan.io/extrinsic/0x7bd468eeaa0ada089e3415dc327f61aacda59aa824c90a74ce7b19933febed6a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**